src/machines/workflow_machines.rs

```diff
@@ -139,6 +139,8 @@ pub enum WFMachinesError {
     InvalidTransition(String),
     #[error("Invalid cancelation type: {0}")]
     InvalidCancelationType(i32),
+    #[error("Unable to process partial event history because workflow is no longer cached.")]
+    ResetSticky,
```

Comment:
Let's call this `CacheMiss` or something. A little more clear what went wrong rather than the intended outcome.

Reply:
Yep